src/services/findAllReferences.ts

```diff
@@ -285,6 +285,9 @@ namespace ts.FindAllReferences.Core {
             return undefined;
         }
 
+        // If this property is derived from another one, find references on the original property instead.
+        symbol = (symbol as ts.SymbolLinks).syntheticOrigin || symbol;

```

Comment:
There was a check for `if (!symbol.declarations)` which failed for a synthetic symbol, which has no declarations of its own. Removing this check (and another similar one) seems to cause no problems, though.